omod/src/main/webapp/resources/htmlforms/Walkin-Client-Data-Capture-tool.xml

```diff
@@ -0,0 +1,256 @@
+                                        <label>
+                                            <span class="required">*</span>
+                                            Current Regimen
+                                        </label>
+                                        <obs id="current-regimen" conceptId="90315" answerConceptIds="99015,99016,99005,99006,164979,99039,99040,164977,99041,99042,99007,99008,99044,99043,99282,99283,99046,99017,99018,99019,99045,99284,99285,99286,99884,99885,164978,99888,163017,99887,90002" answerLabels="d4T-3TC-NVP,d4T-3TC-EFV,AZT-3TC-NVP,AZT-3TC-EFV,AZT-3TC-DTG,TDF-3TC-NVP,TDF-3TC-EFV,TDF-3TC-DTG,TDF-FTC-NVP,TDF-FTC-EFV,ABC-ddI(250)-LPV/r,ABC-ddI(400)-LPV/r,TDF-3TC-LPV/r,TDF-FTC-LPV/r,ZDV-ddI(250)-LPV/r,ZDV-ddI(400)-LPV/r,AZT-3TC-LPV/r,ABC-ddI-LPV/r,ABC-ddI-NFV,ABC-ddI-SQV/r,ZDV-ddI-LPV/r,AZT-ABC-LPV/r,ABC-ddI-ATV/r,AZT-3TC-ATV/r,ABC-3TC-NVP,ABC-3TC-EFV,ABC-3TC-DTG,ABC-3TC-ATV/r,ABC-3TC-LPV/r,TDF-3TC-ATV/r,Other" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            Parent Health facility
+                                        </label>
+                                        <obs id="parent-health-facility" conceptId="166453" />
+                                        <obs id="district-of-parent-health-facility" labelText="District of Parent Health Facility" conceptId="166454" />
+                                        <label>
+                                            <span class="required">*</span>
+                                            Serial No
+                                        </label>
+                                        <obs id="serial-number" conceptId="1646" />
+                                        <label>Encounter Date</label>
+                                        <encounterDate default="today" disallowMultipleEncountersOnDate="block" />
+                                    </div>
+
+                                    <div class="row">
+                                        <div class="col-md-12">
+                                            <obs id="reason-for-visit-other" labelText="Other Specify" conceptId="164975" />
+                                            <obs id="other-regimen" labelText="Other Regimen" conceptId="99126" />
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+
+                        <div class="card">
+                            <div class="card-body">
+                                <div class="row">
+                                    <div class="col-md-3">
+                                        <label>
```

Comment:
@ODORA0 Make the column larger use col-md-4

Reply:
@ssmusoke Done